disnake/state.py

```diff
@@ -608,8 +608,8 @@ def _get_guild_channel(
         try:
             guild = self._get_guild(int(data["guild_id"]))
         except KeyError:
-            # if we're here, this is a DM channel
-            channel = self._get_private_channel(channel_id)
+            # if we're here, this is a DM channel or an ephemeral message in a guild
+            channel = self.get_channel(channel_id)
```

Comment:
1) Payloads with ephemeral messages contain guild IDs, don't they?
2) Do bots even receive ephemeral messages?

Reply:
1. sadly not.
2. not sure if they receive ephemeral messages from other bots.